candi/bashible/common-steps/all/000_add_node_users.sh.tpl

```diff
@@ -215,6 +215,38 @@ function add_sudoer_group() {
     fi
 }
 
+# $1 - user name
+function add_node_annotation() {
+  local username="$1"
+  local failure_count=0
+  local failure_limit=5
+
+  until bb-kubectl-exec --kubeconfig=/etc/kubernetes/kubelet.conf annotate node $(bb-d8-node-name) node.deckhouse.io/nodeuser="${username}"; do
+    failure_count=$((failure_count + 1))
+    if [[ $failure_count -eq $failure_limit ]]; then
+      bb-log-error "ERROR: Failed to annotate node $(bb-d8-node-name)"
+      break
+    fi
+    bb-log-error "failed to annotate node $(bb-d8-node-name)"
+    sleep 10
+  done
+}
+
+function remove_node_annotation() {
```

Comment:
we can move this functions in 098 step, and use for annotate node in general way

Reply:
for all annotations